### DISCUSSION TITLE
Generating a Mongoose.pdf

### DISCUSSION THREAD

**Discussion by gvanem** [cesanta/mongoose#2510] [Ask questions, get answers]
Hello folks.

I want to scrape the docs from `https://mongoose.ws/documentation/` and save it into a `Mongoose.pdf` using the
built-in *Print | Save to PDF* option in Chrome. The result looks kinda ugly:
![mongoose-pdf](https://github.com/cesanta/mongoose/assets/945271/fd2adf7a-94f7-4263-9f7e-13b83183a9ad)

These *Rate our documentation* showing on every page (shown in Sumatra PDF viewer). I figured a trick was to
block that *Poll* cookie.

But instead I'd like to use Pandoc to generate a local `Mongoose.pdf`. I tried with:
`pandoc --output Mongoose.pdf --highlight-style espresso --verbose --trace https://mongoose.ws/documentation/ 2> pandoc.log`

But here MikTeX failed with `LaTeX Error: Unknown graphics extension: .webp.`

Anyone got a hint?

**Comment by scaprile** [comment]
I think we should use one of the available tools and generate a PDF on every release based on the markdown source.
However, yours is the 2nd request I read in a year and a half, so my guess is that there is no big demand for that and we have limited resources.

**Reply by gvanem** [reply]
> I think we should use one of the available tools and generate a PDF on every release based on the markdown source.

That would be great. BTW, where are the markdown sources for this?

**Reply by scaprile** [reply]
Those belong to our doc site, closed source

**Comment by cpq** [comment]
I think we should generate .pdf and attach as a download as part of a release process.
Every release then would have a relevant reference.

**Reply by scaprile** [reply]
I'll see to get a script and maybe run it as a workflow job.
